test/metabase/events/recent_views_test.clj

```diff
@@ -11,7 +11,7 @@
   [user-id]
   (t2/select-one [:model/RecentViews :user_id :model :model_id]
                  :user_id user-id
-                 {:order-by [[:timestamp :desc]]}))
+                 {:order-by [[:timestamp :desc] [:id :desc]]}))
```

Comment:
Shouldn't we remove the ordering by `[:timestamp :desc]`?

Reply:
Yeah, it'll be the same behavior either way but I guess it's cleaner to just order on ID